webpack/scenes/Subscriptions/Manifest/SCAContainer.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useRef } from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PropTypes from 'prop-types';
+import SimpleContentAccess from './SimpleContentAccess';
+
+const SCAContainer = (props) => {
+  const {
+    isSimpleContentAccessEnabled,
+  } = props;
+  const [enabled, setEnabled] = useState(isSimpleContentAccessEnabled);
+  const hiddenRailsInput = useRef(document.querySelector('#content_access_mode'));
```

Comment:
Could this possibly be given as a prop, or should it? In the context of this container I don't think we care about Rails or whether it's hidden so I'd recommend a name along the lines of 'scaFormInput'

Reply:
I don't think a DOM element could be given as a prop since it's starting out in Ruby, but the name change makes sense to me.  Will update